File: problems/3718/solution.py
class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        rec = [False] * 101
        for n in nums:
            rec[n] = True
        ptr = k
        while ptr <= 100:
            if rec[ptr] == False:
                return ptr
            ptr += k

        return ptr

File: problems/3718/solution.cpp
#include <vector>
using namespace std;

class Solution {
 public:
  int missingMultiple(vector<int>& nums, int k) {
    bool rec[101] = {false};
    for (auto& n : nums) {
      rec[n] = true;
    }
    int ptr = k;
    while (ptr <= 100) {
      if (!rec[ptr]) {
        return ptr;
      }
      ptr += k;
    }
    return ptr;
  }
};